suite/tests/CMakeLists.txt

```diff
@@ -4134,6 +4134,11 @@ if (BUILD_CLIENTS)
       set(tool.drcacheoff.burst_traceopts_nodr ON)
       set(tool.drcacheoff.burst_traceopts_nopost ON)
       torunonly_drcacheoff(burst_traceopts tool.drcacheoff.burst_traceopts "" "" "")
+      set(tool.drcacheoff.windows-timestamps_nodr ON)
+      set(tool.drcacheoff.windows-timestamps_nopost ON)
+      torunonly_drcacheoff(windows-timestamps tool.drcacheoff.windows-timestamps "" "" "")
+      set(tool.drcacheoff.windows-timestamps-expectbase "offline-windows-timestamps")
+      set(tool.drcacheoff.windows-timestamps_timeout 120)
```

Comment:
I can't seem to find the prior comment thread on the timeout value.  Looks like it took 100s in the x64 run. 
 Why does the test take so long to run?  Can we shrink it by reducing the sleep times or shrinking the trace?  Trying to keep the CI closer to 15 mins and each mulit-minute test pushes it longer.

Reply:
Your new test here is by far the longest test in the suite.  Here's x86-64:
```
$ grep 'Test #' x86-64/7_Run\ Suite.txt | sort -k 8 -n
...
2023-08-24T00:54:52.5859811Z 377/470 Test #378: code_api|tool.drcacheoff.burst_threads_counts ....................   Passed    7.02 sec
2023-08-24T00:54:52.5851752Z 376/470 Test #377: code_api|tool.drcacheoff.burst_threads ...........................   Passed    7.15 sec
2023-08-24T00:53:06.4469976Z 333/470 Test #329: code_api|tool.drcacheoff.multiproc ...............................   Passed    8.84 sec
2023-08-24T00:52:37.5857344Z 311/470 Test #310: code_api|tool.drcachesim.multiproc ...............................   Passed   10.29 sec
2023-08-24T00:54:04.7949137Z 366/470 Test #364: code_api|tool.drcacheoff.func_view_heap ..........................   Passed   11.14 sec
2023-08-24T00:54:52.5866791Z 378/470 Test #379: code_api|tool.drcacheoff.burst_malloc ............................   Passed   13.76 sec
2023-08-24T00:50:56.0188972Z 153/470 Test #138: code_api|client.alloc ............................................   Passed   14.82 sec
2023-08-24T00:55:25.4520321Z 381/470 Test #382: code_api|tool.drcacheoff.burst_threadL0filter ....................   Passed   15.01 sec
2023-08-24T00:55:25.4511924Z 380/470 Test #381: code_api|tool.drcacheoff.burst_threadfilter ......................   Passed   15.20 sec
2023-08-24T00:54:52.5874247Z 379/470 Test #380: code_api|tool.drcacheoff.burst_reattach ..........................   Passed   16.40 sec
2023-08-24T00:53:29.5240263Z 346/470 Test #337: code_api|tool.drcacheoff.sysnums .................................   Passed   23.07 sec
2023-08-24T00:52:50.0526383Z 320/470 Test #311: code_api|tool.drcachesim.miss_analyzer ...........................   Passed   25.35 sec
2023-08-24T01:08:42.9138230Z 132/132 Test #370: code_api|tool.drcacheoff.windows-timestamps ......................   Passed  100.06 sec
2023-08-24T00:55:48.3853602Z 406/470 Test #375: code_api|tool.drcacheoff.windows-timestamps ......................   Passed  100.35 sec
```

Reply:
Done. Decreased the sleep time and input.